commit 2379d8de25972e20ce825be02a7cca87facfd58f (refs/changes/06/4706/1)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-08-16T17:12:27-04:00 (1 year, 2 months ago)
    
    joyent/node-manta#347 mmkdir test fails when MANTA_TEST_ROLE is unset

diff --git a/test/mmkdir.test.js b/test/mmkdir.test.js
index e2063cf..b732f49 100644
--- a/test/mmkdir.test.js
+++ b/test/mmkdir.test.js
@@ -85,16 +85,15 @@ if (process.env.MANTA_TEST_ROLE) {
                   });
            });
     });
-}
-
 
-test('cleanup: rm test directory ' + TESTDIR, function (t) {
-    // Sanity checks that we don't `mrm -r` a non-test dir.
-    assert.ok(TESTDIR);
-    assert.ok(TESTDIR.indexOf('node-manta-test') !== -1);
+    test('cleanup: rm test directory ' + TESTDIR, function (t) {
+        // Sanity checks that we don't `mrm -r` a non-test dir.
+        assert.ok(TESTDIR);
+        assert.ok(TESTDIR.indexOf('node-manta-test') !== -1);
 
-    forkExecWait({ argv: [ MRM, '-r', TESTDIR ]}, function (err) {
-        t.ifError(err, err);
-        t.done();
+        forkExecWait({ argv: [ MRM, '-r', TESTDIR ]}, function (err) {
+            t.ifError(err, err);
+            t.done();
+        });
     });
-});
+}
